packages/editor/src/functions/sceneFunctions.tsx

```diff
@@ -74,7 +74,9 @@ export const saveSceneGLTF = async (
 
   const { rootEntity } = getState(EditorState)
 
-  const sceneName = cleanString(sceneFile!.replace('.scene.json', '').replace('.gltf', '')) + '.gltf'
+  const baseSceneName = cleanString(sceneFile!.replace('.scene.json', '').replace('.gltf', '').trim())
```

Comment:
our .scene.json format is over a year old now, we can remove the check for that here too